conda_declarative/apply.py

```diff
@@ -131,5 +137,69 @@ def link(
             neutered_specs=(),
         )
     )
-    handle_txn(txn, prefix, args, False, True)
+
+    handle_txn(
+        unlink_link_transaction=txn,
+        prefix=prefix,
+        args=args,
+        newenv=False,
+        remove_op=False,  # Must be False, unless `args.package_names` is provided
+    )
     return txn
+
+
+def apply(
+    prefix: str | None = None,
+    quiet: bool = False,
+    dry_run: bool = False,
+    lock_only: bool = False,
+    *args,
```

Comment:
```suggestion
    args: argparse.Namespace = None
```

It look like this `args` is passed to the `link` function. In the `link` function, `args` is a `argparse.Namespace`. These should match.